### PR TITLE
Guide on moving a subdirectory package to its own repository

### PR DESCRIPTION
**Pull request by mtfishman**
This is a summary of the process used in #105961 and #106369 to split off a subdirectory package into its own repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +243,13 @@ Technically if you skip the second step things will keep working, because GitHub
 but it is best practice. For this reason, when you try to register a new release, the Julia
 Registrator will complain if the second step is skipped.
 
+### How do I move a subdirectory package to its own repository?
+
+Follow these steps to move a [subdirectory package](https://pkgdocs.julialang.org/v1/managing-packages/#Adding-a-package-in-a-subdirectory-of-a-repository) to its own repository:
+1. Follow Github's documentation on [splitting a subfolder out into a new repository](https://docs.github.com/en/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository).
```
Comment (ericphanson) [inline README.md]:
```suggestion
1. Follow Github's documentation on [splitting a subfolder out into a new repository](https://docs.github.com/en/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository). If you aren't hosting your package on GitHub, most of the steps are still applicable.
```

File: README.md [hunk 2]
```diff
@@ -243,6 +243,13 @@ Technically if you skip the second step things will keep working, because GitHub
 but it is best practice. For this reason, when you try to register a new release, the Julia
 Registrator will complain if the second step is skipped.
 
+### How do I move a subdirectory package to its own repository?
+
+Follow these steps to move a [subdirectory package](https://pkgdocs.julialang.org/v1/managing-packages/#Adding-a-package-in-a-subdirectory-of-a-repository) to its own repository:
+- Follow Github's documentation on [splitting a subfolder out into a new repository](https://docs.github.com/en/get-started/using-git/splitting-a-subfolder-out-into-a-new-repository).
+- [Check that the new repository contains all registered versions of the package](https://github.com/JuliaRegistries/General/blob/master/CONTRIBUTING.md#appendix-checking-if-a-repository-contains-all-registered-versions-of-a-package).
+- [Update the URL of the package in the General registry to point towards the new repository](https://github.com/JuliaRegistries/General#how-do-i-transfer-a-package-to-an-organization-or-another-user).
```
Comment (ericphanson) [inline README.md:251]:
```suggestion
- [Update the URL of the package in the General registry to point towards the new repository](https://github.com/JuliaRegistries/General#how-do-i-transfer-a-package-to-an-organization-or-another-user). Be sure to include the printed output from the second step in your pull request, so the reviewer can see that the new repository indeed contains all registered versions fo the package.
```

**Comment by mtfishman** [comment]
Following the guide linked to by the last step might be a bit confusing since https://github.com/JuliaRegistries/General#how-do-i-transfer-a-package-to-an-organization-or-another-user also includes the step "Use the [GitHub transfer option][github-transfer] in the settings." but maybe it is clear that isn't needed here.

**Comment by ericphanson** [comment]
Maybe just write the URL step again?

> Manually, in the General edit the repo URL in package's Package.toml file (e.g [E/Example/Package.toml](https://github.com/JuliaRegistries/General/blob/master/E/Example/Package.toml#L3)), and remove the `subdir` line.

**Comment by mtfishman** [comment]
I updated the wording in both sections, hopefully that is clearer.

**Review by ericphanson — APPROVED** [review]
LGTM, thanks!

**Pull request merged**